armeria/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>io.zipkin.brave.example</groupId>
+  <artifactId>brave-example-armeria-java</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>brave-example-armeria</name>
+  <description>Tracing Example: Armeria</description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+
+    <armeria.version>1.1.0</armeria.version>
+    <brave.version>5.12.6</brave.version>
```

Comment:
Just a question: Is it intentional that we did not define a parent POM but kept this property here?

Reply:
the projects are disconnected intentionally to reduce temptation to make a soupy classpath, address different build requirements and future ones (like eventual graal or otherwise in some case)

All that said, it is true the one thing I'd also like to have is the brave version updated once.. especially as we have CI across everything now

Reply:
seems easiest to make a brave-version.properties? https://stackoverflow.com/questions/849389/how-to-read-an-external-properties-file-in-maven

Reply:
will do this in another PR!

Reply:
https://github.com/openzipkin/brave-example/pull/49